Formula/qgis3.rb

```diff
@@ -120,6 +121,13 @@ def pour_bottle?
   depends_on "poppler"
   depends_on "gnu-sed"
   depends_on "exiv2"
+  depends_on "liblas"
+  depends_on "netcdf"
+  depends_on "pdal"
+  depends_on "openvpn"
```

Comment:
I think these dependencies should be optional:
- liblas
- netcdf
- pdal
- openvpn
- szip
- hdf5

Reply:
Done! the dependencies are already optional